### DISCUSSION TITLE
Super-silly beginner's question: What does the q keystroke do and how to get back to NORMAL mode?

### DISCUSSION THREAD

**Discussion by halloleo** [andmarti1424/sc-im#766] [Q&A]
I'm just staring out with sc-im and it looks super cool.

However because I'm not too used to the vi/vim style `:q` for quitting, I'm sometimes just hitting `q` _without_ the colon.

Then sc-im seems to be in a strange state, which I get it out of after a few (random) keystrokes, but this is not satisfatory. 

What's the best way to get back to NORMAL mode (and then via colon to COMMAND mode) once I typed accidentally `q` in NORMAL mode?

**Comment by andmarti1424** [comment]
ESC key

**Reply by halloleo** [reply]
Aha, cool. So Esc gets me out of it, excellent. ut what does `q` alone gets me into?

**Comment by andmarti1424** [comment]
nothing by default.

**Reply by halloleo** [reply]
Thanks @andmarti1424 for being so patient!

However I don't understand it: When `q` does nothing, why can't I press `:` `q` afterwards? Why do I ahvet to press `ESC` first? Feels to me the plain `q` _does_ change _something_ - maybe the mode? 

Sorry for these beginner's questions, but I really don't get it...
